### ISSUE TITLE
[React Native Gallery Preview]: Screen reader is not announcing the success message after activating the ‘Paste text from the Clipboard' button.

### ISSUE THREAD

**Issue by Nitin-100** [microsoft/react-native-gallery#649]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
AT Used: Narrator

Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘Clipboard’ button present under “System” group and invoke it.
Navigate to ‘Copy text to the Clipboard’ button and listen to narrator announcement.
Actual Result:
Upon activating 'Paste text from Clipboard' button, narrator fails to announce the success information to the user. When user invoke the ‘Paste text from Clipboard' button, narrator stays on mute.
 
Note: Same issue is observed upon activating ‘Copy text to the Clipboard' button.
 
Expected Result:
Narrator should announce the success information to the user after activating the 'Paste text from Clipboard' button and it should announce like "Text pasted from clipboard". So that, user can understand clearly.
 
User Impact:
Users with visual impairments who rely on screen reader will not be able to understand if the narrator is not announcing the status message after activating the button.
 
MAS Reference Link: [MAS 4.1.3 – Status Messages](https://aka.ms/MAS4.1.3)

Trap ID: [1.9 - Feedback Failure](https://aka.ms/FeedbackFailure)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
